src/main/java/utils/methods/ReusableMethods.java

```diff
@@ -0,0 +1,30 @@
+package utils.methods;
+
+import utils.variables.SessionVariables;
+
+import java.util.Random;
+
+import static utils.variables.SessionVariableManager.getSessionVariable;
+import static utils.variables.SessionVariables.VAR_USER_ID;
+
+public class ReusableMethods {
+
+    public int generateRandomInt(int min, int max){
+        Random randomId = new Random();
+        return randomId.ints(min, (max+1)).findFirst().getAsInt();
+    }
+
+    public String getRandomStringValue(){
+        Random random = new Random();
+        int randomInt = random.nextInt();
+        return Integer.toString(randomInt);
+    }
+
+    public String setSessionVariableAsExpected(String expectedString, SessionVariables sessionVariable) {
```

Comment:
the name is confusing, someone can understand this will set the session variable, which it doesn't.